consensus/enclave/api/src/lib.rs

```diff
@@ -187,6 +187,14 @@ pub struct TxContext {
 
 pub type SealedBlockSigningKey = Vec<u8>;
 
+/// PublicAddress is not serializable with serde currently, and rather than pollute
+/// dependencies, we simply pass the View and Spend public keys as RistrettoPublic.
+#[derive(Clone, Debug, Default, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize)]
+pub struct FeePublicKey {
+    pub fee_spend_public_key: RistrettoPublic,
```

Comment:
Nit: The fee_ prefix is redundant since the struct name already contains it.

Reply:
Good point - fixed